pkgs/development/libraries/libseccomp/default.nix

```diff
@@ -25,13 +25,13 @@ stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "High level library for the Linux Kernel seccomp filter";
-    homepage    = "https://github.com/seccomp/libseccomp";
-    license     = licenses.lgpl21;
-    platforms   = platforms.linux;
+    homepage = "https://github.com/seccomp/libseccomp";
+    license = licenses.lgpl21;
+    platforms = platforms.linux;
     badPlatforms = [
       "alpha-linux"
-      "riscv64-linux" "riscv32-linux"
```

Comment:
I only see that they added support for riscv64-linux. (https://github.com/seccomp/libseccomp/blob/5cd9059618a0810ee47c21e6b44c5a876b75e23d/src/arch-riscv64.c) Where do you see riscv32-linux support?

Reply:
You're right, I readded the architecture there: https://github.com/NixOS/nixpkgs/pull/96337